### ISSUE TITLE
Slider Spot Check Sliders in sims published with Interactive Description

### ISSUE THREAD

**Issue by terracoda** [phetsims/qa#983]
This morning I noticed that the sliders in the published versions of GFL (regular) were not working properly with mobile VoiceOver iOS 15, nor with VoiceOver on MacOS Monterey 12.6.2. Sam Reid tested with Ventura 13.4 and the sliders worked.

When QA has time, could you test the sliders of a few published sims that have the Interactive Description feature to see what version of MacOS and iOS is required for sliders to work with iOS VoiceOver and MacOS VoiceOver?

I previously thought with iOS 15 everything worked but that was not the case with iOS 15 this morning, and I was not aware that sliders were no longer working on MacOS 12.6.2. Maybe MacOS 13 is required?

Sims with sliders and Interactive Description to spot check are:

Sliders working on mobile VoiceOver?
- [x] RaP
- [x] JT
- [x] GFLB
- [x] Molarity
- [x] OL
- [x] RIAW

Sliders working on MacOS VoiceOver?
- [x] RaP
- [x] JT
- [x] GFLB
- [x] Molarity
- [x] OL
- [x] RIAW

**Comment by terracoda** [comment]
I would suggest testing with what you have if later than MacOS 12.6.2 and iOS 15, and then updating if the sliders are not working and what you are currently running is not the latest.

Please make not of versions :-)

**Comment by terracoda** [comment]
@KatieWoe rightly pointed out that screen reader support is limited to the latest OS version, so please just spot check the sliders on the latest iOS and MacOS's and make note of those versions.

The goal is to verify that interactive description and alt input work as expected on sliders interactions in the published sims.

**Comment by terracoda** [comment]
@KatieWoe mentioned a known issue with Ratio and Proportion's hand slider https://github.com/phetsims/ratio-and-proportion/issues/530 

It would be good to know if the latest version of MacOS has addressed missing object responses, too.

**Comment by KatieWoe** [comment]
Jaws and NVDA seemed ok. iOS just updated today, so I'll be doing a more in depth check on it tomorrow.

**Comment by terracoda** [comment]
@KatieWoe, since you are poking around, do you think you can spot check for this https://github.com/phetsims/ratio-and-proportion/issues/530?

And check the Light Source on/off Switch and Pause/Play buttons in MAL. We unpublished mobile Interactive Description for that sim a while back because toggle buttons got totally broken. Some toggle buttons are working now, but I am not sure about those 2 - the Light Source Switch is essential to run the sim.

**Comment by KatieWoe** [comment]
MAL still seems to be broken.

**Comment by KatieWoe** [comment]
Sliders looked ok after quick checks on iPadOS 17.1.1 + VO

**Comment by Nancy-Salpepi** [comment]
The issue I reported in https://github.com/phetsims/ratio-and-proportion/issues/530 looks fixed with macOS 14 and Safari 17.

**Comment by Nancy-Salpepi** [comment]
Sliders seemed to be working correctly with macOS14 + safari 17 + VO

**Comment by terracoda** [comment]
MAL´s Light Source “toggle” is broken because it has role switch. It’s likely that VoiceOver hasn’t fixed that yet. But it is good that they fixed the the general toggle button role.

**Comment by KatieWoe** [comment]
Closing
